Makefile

```diff
@@ -0,0 +1,2 @@
+all:
+	docker build -t collabtest/collabtest . && docker run -it --rm -p 8080:8080 collabtest/collabtest
```

Comment:
we probably want to run the container with a name `--name collabtest` and also delete any running containers with that name. something like `docker build -t collabtest/collabtest . && (docker rm -f collabtest; docker run -it --rm -p 8080:8080 --name collabtest collabtest/collabtest)